hyperliquid/exchange.py

```diff
@@ -76,19 +79,19 @@ def order(
             {
                 "type": "order",
                 "grouping": grouping,
-                "orders": [order_spec_to_order_wire(order_spec)],
+                "orders": [order_spec_to_order_wire(order_spec) for order_spec in order_specs],
             },
             signature,
             timestamp,
         )
 
-    def cancel(self, coin: str, oid: int) -> Any:
+    def cancel(self, coin: str, oids: List[int]) -> Any:
```

Comment:
I feel similarly that we should have a separate cancels method and that should take a list of coin, oid pairs instead of just a list of oids